bears/general/FilenameBear.py

```diff
@@ -26,6 +27,8 @@ class FilenameBear(LocalBear):
                                   }
 
     def run(self, filename, file,
+            max_len_path_unix: int=4096,
+            max_len_path_windows: int=260,
```

Comment:
Ah interesting, separating this for different systems. I'm just considering if it still can be made a single settings, I'm thinking of
```
max_path_length = 4096  # <-- affect all systems
max_path_length = {'windows': 100, 'unix': 100} # <-- different settings for different OSes, but this is probably too complicated from user perspective
```
However I think this can be a new issue.

Reply:
What do you think about such situation:
When we have different limits on different systems, then when project is moved to other system, then this code wouldn't be useful. Because on first system limit was 260, but on second system is only 255. 
In my opinion setting the minimum of these limits as default should resolve it.